sig-scheduling/2020-annual-report.md

```diff
@@ -0,0 +1,125 @@
+# 2020 SIG-Scheduling Annual Report
+
+## Operational
+
+**How are you doing with operational tasks in [sig-governance.md]?**
+    
+- **Is your README accurate? have a CONTRIBUTING.md file?**  
+    Yes. Developer-oriented guides are under [contributors/devel/sig-scheduling].
+
+- **All subprojects correctly mapped and listed in [sigs.yaml]?**  
+    Yes.
+
+- **What's your meeting culture? Large/small, active/quiet, learnings? 
+Meeting notes up to date? Are you keeping recordings up to date/trends 
+in community members watching recordings?**  
+    Our meetings tends to be small and quiet. The agenda usually consists of items suggested
+    by users, and debatable items that need a consensus during issues/PR reviews.
+    The meeting notes and recordings are up to date.
+
+**How does the group get updates, reports, or feedback from subprojects? Are there any 
+springing up or being retired? Are OWNERS files up to date in these areas?**
+
+Owners from (active) subprojects introduce the latest development, and sometime demonstrate
+cool features.
+OWNER files in k/k are not that up to date. We may need a cleanup.
```

Comment:
is this something contribex can help with?

Reply:
I think we are fine, we can send a PR to remove inactive members and tag them on that PR.

Reply:
I suppose it's a common issue across all sigs. It'd be neat if we can come up with a script auto-identifying the inactive maintainers/reviewers:

- for a particular SIG, not participated in any PR reviews in the last 12 (or 18) months
- a single bare "/lgtm" or "/approve" doesn't count as a legal review

Reply:
ok